### DISCUSSION TITLE
No data is showing and "Forbidden" msg appears on a screen in Public access section

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#2002] [Help] [answered]
Hi all,

I am trying to setup a screen/page with Public access. The page has a Data Provider and Repeater. 
I use a url like _`http:/my_domain/app_5043e800874e4143b6e606194baed597#/oil/T4897`_

No data is showing and  "Forbidden" message appears when I access the page from another computer/browser without BB Log in,
Can you please advise what I missed? How to get the page working without log in?

**Comment by joebudi** [comment, marked as answer]
Hey @sergey-dme 

Are you using an internal table? If so, have you clicked the 'manage access' button and set the dropdowns to public?
![Screenshot 2021-07-16 at 13 51 46](https://user-images.githubusercontent.com/49767913/125950582-1baaeb0d-8764-4692-ae59-987e926d54bd.png)

**Reply by joebudi** [reply]
At least for read*

**Reply by sergey-dme** [reply]
@joebudi Thank you very much Joe! I missed the "Manage Access" settings. Now it works!
